### PR TITLE
Allow custom keystore and truststore types

### PR DESCRIPTION
**Pull request by csunwold**
Using the default provided Keystore type is not always desirable, especially when you want to use a custom Keystore. This makes the keystore and truststore types configurable. This is most useful for the JDBC driver. If the keystore or truststore types are not configured, the default system provided types continue to be used.

### CODE REVIEW

File: presto-cli/src/main/java/io/prestosql/cli/ClientOptions.java [hunk 1]
```diff
@@ -77,12 +77,18 @@
     @Option(name = "--keystore-password", title = "keystore password", description = "Keystore password")
     public String keystorePassword;
 
+    @Option(name = "--keystore-type", title = "keystore password", description = "Keystore password")
```
Comment (Lewuathe) [inline presto-cli/src/main/java/io/prestosql/cli/ClientOptions.java:80]:
```suggestion
    @Option(name = "--keystore-type", title = "keystore type", description = "Keystore type")
```

File: presto-docs/src/main/sphinx/installation/jdbc.rst [hunk 2]
```diff
@@ -100,9 +100,11 @@ Name                                   Description
 ``SSLKeyStorePath``                    The location of the Java KeyStore file that contains the certificate
                                        and private key to use for authentication.
 ``SSLKeyStorePassword``                The password for the KeyStore.
+``SSLKeyStoreType``                    The type of the KeyStore.
 ``SSLTrustStorePath``                  The location of the Java TrustStore file to use.
                                        to validate HTTPS server certificates.
 ``SSLTrustStorePassword``              The password for the TrustStore.
+``SSLTrustStoreType``                  The type of the TrustStore..
```
Comment (Lewuathe) [inline presto-docs/src/main/sphinx/installation/jdbc.rst:107]:
```suggestion
``SSLTrustStoreType``                  The type of the TrustStore.
```

File: presto-docs/src/main/sphinx/installation/jdbc.rst [hunk 3]
```diff
@@ -100,9 +100,11 @@ Name                                   Description
 ``SSLKeyStorePath``                    The location of the Java KeyStore file that contains the certificate
                                        and private key to use for authentication.
 ``SSLKeyStorePassword``                The password for the KeyStore.
+``SSLKeyStoreType``                    The type of the KeyStore.
```
Comment (Lewuathe) [inline presto-docs/src/main/sphinx/installation/jdbc.rst:103]: We can put the default key store name `JKS` or just saying ["The default key store is provided by the Java security property files"](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/security/KeyStore.html#getDefaultType()).

**Comment by csunwold** [comment]
Thanks @Lewuathe for the feedback. I've addressed all comments.

**Comment by electrum** [comment]
Thanks!

**Pull request merged**